hack/charts/installer/rolebased/README.md

```diff
@@ -1,5 +1,49 @@
-things that need to be converted to helm:
-* AWS_DEFAULT_SAGEMAKER_ENDPOINT (TODO)
+# Amazon SageMaker Operator for Kubernetes Role-Based Installer
 
-Notes:
-* `kustomize build config/helm` -> has names that are actually variable references. We then replace those using helm.
+Amazon SageMaker Operator for Kubernetes Helm charts for role-based installation
+
+## Prerequisites
+
+* Kubernetes >= 1.13
+
+## Installing the Chart
+
+Clone the existing repository:
+
+```bash
+git clone https://github.com/aws/amazon-sagemaker-operator-for-k8s.git
+```
+
+Navigate to the helm chart directory and edit the values in the configuration file:
+
+```bash
+cd amazon-sagemaker-operator-for-k8s/hack/charts/installer
+vim rolebased/values.yaml
+```
+
+The [configuration](#configuration) section below lists the parameters that can be configured for installation.
+
+Install the operator using the following command:
+
+```bash
+kubectl create namespace sagemaker-k8s-operator-system
+helm install --namespace sagemaker-k8s-operator-system sagemaker-operator rolebased/
+```
+
+## Uninstalling the chart
+
+To uninstall/delete the operator deployment:
+
+```bash
+helm delete --purge sagemaker-operator
+```
+
+## Configuration
+
+The following table lists the configurable parameters for the chart and their default values.
+
+Parameter | Description | Default
+--- | --- | ---
+`rolesArn` | The EKS service account role ARN  | `<PLACEHOLDER>`
+`image.repository` | image repository | `957583890962.dkr.ecr.us-east-1.amazonaws.com/amazon-sagemaker-operator-for-k8s`
+`image.tag` | image tag | `<VERSION>`
```

Comment:
Should we leave out or complete? I think instead of providing an incomplete table we should just refer them to the spec?

Reply:
This is complete for the installer chart. These are the only values in the `values.yaml` file they are able to modify.

Reply:
I left the README out of the specs since I would hope users refer to the documentation for the specification rather than a possibly out-of-date README.

Reply:
I see, I got confused between charts. 

> I left the README out of the specs since I would hope users refer to the documentation for the specification rather than a possibly out-of-date README.

Can you restate? Do you mean to say that you did not include these installation steps in the spec itself?

Reply:
I meant I did not include the `README.md` file in the training/hpo/batch charts since they use the same spec as the YAML charts, so it would be duplication of information that we would need to keep up-to-date.